### ISSUE TITLE
Bulkrax doesn't complete/update metadata

### ISSUE THREAD

**Issue by mlhale7** [notch8/utk-hyku#685]
## Summary
- A spreadsheet was uploaded through bulkrax to remediate metadata associated with the volunteer voices collection on Oct. 17th. While a green check mark appeared and the status is "Complete", no records were updated. It's important that metadata can be easily updated with associated Solr updates running in the future (for updating metadata with potentially harmful language and for filling out minimal records etc.). I'll test other collections to see if how widespread this is.

![Image](https://github.com/user-attachments/assets/05868dab-e8e3-4bc4-a237-56a79f51de26)

## Acceptance Criteria
- [x] Metadata is visibly updated in the UI following a bulkrax metadata update

## Testing Instructions
-  Here's the spreadsheet I uploaded in case it's useful for testing ([volvoices-Alyssa (1).csv](https://github.com/user-attachments/files/17721220/volvoices-Alyssa.1.csv))

**Comment by aprilrieger** [comment]
I see that you've created an importer to update a metadata field, but it's unclear which specific metadata fields were updated and which ones I should focus on in relation to this CSV.

For future tickets, it would be greatly helpful to include the specific fields you’ve updated and the expected changes. This additional context will help us quickly understand and address the reported issue more efficiently. Thank you!

**Comment by mlhale7** [comment]
Hi @aprilrieger - thanks for this. A number of fields were updated, but the biggest changes occurred in the keyword, subject, and spatial fields. As an example, I'll illustrate the changes that were made to a single record - https://digitalcollections.lib.utk.edu/concern/images/1f82f04d-2d2f-41ac-9473-8d8d40a2656a?locale=en Because these records were often derived from MARC records, a lot of the keywords have unnecessary trailing periods (that make it not possible for them to be grouped with like terms in a facet). Additionally, when possible, we've substituted out uncontrolled keywords for subjects with URIs. Complex headings that include both topical and geographic values were also separated out into the appropriate fields.

So the current record on Hyku has the following for keywords:

Paris (Tenn.)--Social life and customs
Modern times in Tennessee
Nature and the Environment
African-Americans
Paris (Tenn.)--History

With the updated import sheet we wanted the following result:

Subjects: http://id.loc.gov/authorities/subjects/sh85001932 (African Americans) | http://id.loc.gov/authorities/subjects/sh85080593 (Manners and customs) | http://id.loc.gov/authorities/subjects/sh85061212 (History)
Spatial: http://id.loc.gov/authorities/names/n82120713 (Paris (Tenn.))
Keywords: Modern times in Tennessee | Nature and the Environment

I've included the strings in parentheses for clarity, though obviously they can't be used for fields that require URIs. My concern here is that the importer itself never went through in the dashboard and no records were processed. I'm going to try reimporting next week with just a few changes to one field and see that things go through and then try to make changes on a larger scale. For us typically remediating a collection involves lots of changes rather than just a few.

**Comment by laritakr** [comment]
@mlhale7 can you provide an import file for the original data as well? I tried to import data and then update it, and didn't get any errors, so I am wondering if it's something specific to this data set.

**Comment by mlhale7** [comment]
@laritakr - I'm out until Tuesday due to daycare being closed on Monday. By original data, do you simply mean the spreadsheet(s) we used to ingest to Hyku initially? It looks like we used two back in January 2023 to add this collection. I've shared them here. I do think there were some errors initially that needed to be navigated, but I'd need to look at this more.

[volvoices_images2_just_filesets_and_attachments.csv](https://github.com/user-attachments/files/18804672/volvoices_images2_just_filesets_and_attachments.csv)

[volvoices_images2_full_with_collections.csv](https://github.com/user-attachments/files/18804673/volvoices_images2_full_with_collections.csv)

**Comment by laritakr** [comment]
Yes that's what I was referring to, thanks. I will let you know if I need more info than this.

**Comment by josh-morgan117** [comment]
@mlhale7 Did your import update correctly?

**Comment by mlhale7** [comment]
@josh-morgan117 - I'll work on re-importing before next Tuesday (4/1). I'll be missing our meeting that day as I'm teaching a class during that time slot.

**Comment by mlhale7** [comment]
This is still in the process of running, but it is definitely actually updating records. It is doing a lot of things that we don't want (still using random languages other than English for places etc. and duplicating resource type, see https://dc.utk-hyku-production.notch8.cloud/concern/images/2c062d41-a5fc-4f92-9b89-c8f1d83fb950?locale=en). The importer is called "volvoices_retry." I believe most of the issues I'm seeing are beyond the scope of this ticket and are mentioned in other tickets. When all entries have been processed on Wednesday morning (3/26) I'll double check the records and close this if I see the intended changes present (even though problems will still persist).

**Comment by mlhale7** [comment]
I'm now getting an Ajax error when refreshing to see the entries processed. 266 out of the 1956 went through.

![Image](https://github.com/user-attachments/assets/ba788884-d6b0-4a7c-8185-8f171f16bced)

**Comment by mlhale7** [comment]
The intended subject updates definitely have gone through. I'll investigate to see if I need to add tickets or comment on existing tickets for the resource type and publication place issues I'm noticing. I'll also double check the import spreadsheet for errors.

**Comment by mlhale7** [comment]
Noting that the resource type issue appears to be due to both `resource_type` and `resource_type_local` being used between the initial import (local) and update. I'll rerun the importer with a blank `resource_type_local`.
